cmd/dep/status.go

```diff
@@ -956,6 +1123,48 @@ outer:
 	return hasMissingPkgs, 0, errInputDigestMismatch
 }
 
+// basicOutputAll takes an outputter, a sorted project list, and a map of ProjectRoot to *BasicStatus and
```

Comment:
Hi @zjs,

What do you mean a sorted project list? Is that comment copied from elsewhere? We should maintain the order the solver provides.

Reply:
This is based on existing comments from where the argument is defined.

https://github.com/akutz/dep/blob/5f52a57c47b1a9e5714736c0d266bd130154590c/cmd/dep/status.go#L872-L878

Essentially, I'm just trying to express that this function assumes that the supplied input order is what you'd like the output order to be. I'll reword this to actually say that.

Reply:
Hi @zjs,

Understood. I just didn't see anywhere you added new sorting, but I wanted to make sure since the order is so specific with which to begin.